uktrade/data-hub-frontend
# Chart component for new dashboard...

## Description of change
This is the first stage of adding a chart component into the new dashboard that will reflect how many investment projects you have. 

Note: Don't care too much about styling for now as we are not sure if this will resemble the final component. The styling I have added is loosely based on the wireframe until we know more from user research.

## Test instructions
- Run storybook
- Run the new dashboard, see #3248 for instructions

## Screenshots
![Screenshot_2021-01-20_at_10 46 47](https://user-images.githubusercontent.com/10154302/106255807-664a2280-6212-11eb-8029-966eabc31f3e.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
